tests/ha/multi_pod_restart/test_multi_server_pod_restart.py

```diff
@@ -0,0 +1,163 @@
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+"""
+HA test suite for Multiple data Pod restart
+"""
+
+import logging
+import secrets
+import time
+from time import perf_counter_ns
+
+import pytest
+
+from commons.helpers.health_helper import Health
+from commons.helpers.pods_helper import LogicalNode
+from commons.utils import assert_utils
+from config import CMN_CFG
+from libs.di.di_mgmt_ops import ManagementOPs
+from libs.ha.ha_common_libs_k8s import HAK8s
+from libs.s3.s3_rest_cli_interface_lib import S3AccountOperations
+from libs.csm.csm_interface import csm_api_factory
+
+# Global Constants
+LOGGER = logging.getLogger(__name__)
+
+
+class TestMultiDataPodRestart:
+    """
+    Test suite for Multiple Data Pod Restart
+    """
+
+    @classmethod
+    def setup_class(cls):
+        """
+        Setup operations for the test file.
+        """
+        LOGGER.info("STARTED: Setup Module operations.")
+        cls.num_nodes = len(CMN_CFG["nodes"])
+        cls.username = []
+        cls.password = []
+        cls.node_master_list = []
+        cls.hlth_master_list = []
+        cls.node_worker_list = []
+        cls.ha_obj = HAK8s()
+        cls.csm_obj = csm_api_factory("rest")
+        cls.random_time = cls.s3_clean = cls.test_prefix = cls.test_prefix_deg = None
+        cls.s3acc_name = cls.s3acc_email = cls.bucket_name = cls.object_name = cls.node_name = None
+        cls.restore_pod = cls.deployment_backup = cls.deployment_name = cls.restore_method = None
+        cls.qvalue = cls.kvalue = cls.nvalue = None
+        cls.mgnt_ops = ManagementOPs()
+        cls.system_random = secrets.SystemRandom()
+
+        for node in range(cls.num_nodes):
+            cls.host = CMN_CFG["nodes"][node]["hostname"]
+            cls.username.append(CMN_CFG["nodes"][node]["username"])
+            cls.password.append(CMN_CFG["nodes"][node]["password"])
+            if CMN_CFG["nodes"][node]["node_type"] == "master":
+                cls.node_master_list.append(LogicalNode(hostname=cls.host,
+                                                        username=cls.username[node],
+                                                        password=cls.password[node]))
+                cls.hlth_master_list.append(Health(hostname=cls.host,
+                                                   username=cls.username[node],
+                                                   password=cls.password[node]))
+            else:
+                cls.node_worker_list.append(LogicalNode(hostname=cls.host,
+                                                        username=cls.username[node],
+                                                        password=cls.password[node]))
+
+        cls.rest_obj = S3AccountOperations()
+
+    def setup_method(self):
+        """
+        This function will be invoked prior to each test case.
+        """
+        LOGGER.info("STARTED: Setup Operations")
+        self.random_time = int(time.time())
```

Comment:
I think we can remove references of random_time as we are not using it anywhere.
Need to check for other class variables also.

Reply:
Ok will remove this and this is generic I have kept, so once we have the whole file ready we might be able to remove unused